### ISSUE TITLE
TOFU Optimal Trainer Configuration for Llama 3.1 8B

### ISSUE THREAD

**Issue by HumzahM** [locuslab/open-unlearning#141]
Hi,
Do you have the optimal trainer configurations for Llama 3.1 8B on TOFU (specifically forget10?) I figured the provided ones are for Llama 3.2 1B since the provided configurations result in total utility loss for 8B.
Thanks 

<img width="1613" height="398" alt="Image" src="https://github.com/user-attachments/assets/aa490c2c-e513-4251-93e2-5b109b23fb7e" />

**Comment by Dornavineeth** [comment]
Sorry, we don't have the configs. But we have the models with different hyperparameter sweeps in this [collection](https://huggingface.co/collections/open-unlearning/tofu-unlearned-models-6860f6cf3fe35d0223d92e88).
The important hyperparameters, such as epoch, learning rate, alpha, etc, can be figured out from the names in the collection.
You might need to do a similar sweep for the 8b model.
